static/css/components/mybooks-menu.less

```
@@ -114,12 +133,24 @@
   }
 
   .li-count {
+    position: relative;
     float: right;
     font-size: .9em;
     margin-top: 1px;
     color: @count-blue-grey;
     margin-left: 4px;
   }
+
+  /* stylelint-disable selector-max-specificity */
+  .li-count.chevron:after {
+    content: "\203A";
+    margin-left: 5px;
+    display: inline-block;
+    transform: scale(2.0);
+    position: absolute;
+    top: .2em;
```

Small style tweaks. This makes the chevron a little lighter and better centered.

```suggestion
    margin: 0 5px;
    position: absolute;
    font-size: 1.5em;
    font-weight: normal;
    top: 50%;
    transform: translateY(-50%);
    right: 0;
    height: 100%;
```